core/embed/rust/src/ui/model_tt/bootloader/confirm.rs

```diff
@@ -98,17 +98,49 @@ impl<'a> Component for Confirm<'a> {
             Point::zero(),
             Point::new(WIDTH, BUTTON_AREA_START),
         ));
-        let icon_height = if let Some(icon) = self.icon {
-            icon.toif.height()
+
+        let content_area_start = if self.icon.is_some() {
+            40 + CONTENT_PADDING
         } else {
-            0
+            58
```

Comment:
These numbers could be constants

Reply:
https://github.com/trezor/trezor-firmware/commit/ec0f08b337177b98bd9cae8aedae48ebbaab2618